sspl_test/functional_tests/v2/ipmi_simulator/ipmi_mock_data/disk.txt

```diff
@@ -0,0 +1 @@
+HDD 0 Status     | F0h | cr  |  4.1 | Drive Present
```

Comment:
Maintain this ipmi_simulator/ under sspl_test dir directly so we dont need to duplicate this code unnecessarily.